### ISSUE TITLE
Player also moves when flying camera moves

### ISSUE THREAD

**Issue by Moonwards** [moonwards1/Moonwards-Virtual-Moon#930]

**Comment by Zach777** [comment]
@Moonwards I do not get any movement from the avatar when using free camera while walking. I do get movement in free camera when in human flight mode though (M key)

**Comment by Moonwards** [comment]
@Zach777 yeah I see that now. So this isn't important, I'll move it to the backlog column and switch it's priority to low. 

It came up because when making video it can occasionally become an issue, but I can work around it.
